static/app/components/issues/groupList.tsx

```diff
@@ -51,10 +51,11 @@ export type GroupListColumn =
 
 type Props = WithRouterProps & {
   api: Client;
-  endpointPath: string;
   orgSlug: string;
-  query: string;
+  queryParams: Record<string, number | string | string[] | undefined | null>;
   customStatsPeriod?: TimePeriodType;
+  // It defaults to `/organizations/${orgSlug}/issues/`
+  endpointPath?: string;
```

Comment:
Prefer using docblock style comments for these

```suggestion
  /**
   * Defaults to `/organizations/${orgSlug}/issues/`
   */
  endpointPath?: string;
```